app/agents/text/automatic/utils/tools.py

```
@@ -0,0 +1,28 @@
+            shop_id=shop_id,
+            user_email=user_email,
+            session_id="debug"
+        )
+
+        return {
+            "tools_count": len(tools_schema.standard_tools),
+            "tool_names": list(tool_functions.keys()),
+            "tools": [{"name": tool.name, "description": tool.description}
+                     for tool in tools_schema.standard_tools]
+        }
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)